openshift-knative-operator/pkg/monitoring/helpers.go

```diff
@@ -270,3 +270,13 @@ func getRBACManifest() (mf.Manifest, error) {
 	rbacManifest, err := mf.NewManifest(rbacPath)
 	return rbacManifest, err
 }
+
+func KubeRbacProxyOverridesOnly(overrides []base.WorkloadOverride, deployments sets.String) []base.WorkloadOverride {
```

Comment:
I wondered what is the meaning of `Only` in `KubeRbacProxyOverridesOnly`, And I understood as "override the resource in the deployment "only" which has kube-rbac-proxy". Maybe it is better to add function comment.

Reply:
It means only consider the deployment override that has a container name `kube-rbac-proxy`. I will add a  comment.